README.md

```diff
@@ -106,6 +107,11 @@ module "my_rds_instance" {
     subnets = ["${aws_subnet.example.*.id}"] # see above
     rds_vpc_id = "${module.vpc}"
     private_cidr = "${var.private_cidr}"
+
+    tags {
```

Comment:
Nope. It's not `=`. Check i.e. `https://www.terraform.io/docs/providers/aws/r/instance.html` pls.